### PR TITLE
Update suggested hardware to match current existing available flavors

### PR DESCRIPTION
**Pull request by apolinario**

### CODE REVIEW

**Review by Wauplin — APPROVED** [review]
Thanks for updating this! LGTM but let's wait for a second review from @co42 or @christophe-rannou who's managing the hardware flavors.

Also, would you mind updating this list in [huggingface.js](https://github.com/huggingface/huggingface.js/blob/99bbf1fd3206680e78ace758933785f38bb72c27/packages/hub/src/types/public.ts#L21) and [huggingface_hub](https://github.com/huggingface/huggingface_hub/blob/c32602ac03308812595b271233140710f1865be8/src/huggingface_hub/_space_api.py#L48) as well to be consistent? :pray:

File: docs/hub/spaces-config-reference.md
Comment (christophe-rannou) [inline docs/hub/spaces-config-reference.md:36]:
```suggestion
	`"l4x4"`, `"a10g-small"`, `"a10g-large"`, `"a10g-largex2"`, `"a10g-largex2"`,
```
duplicate

**Review by christophe-rannou — APPROVED** [review]

**Review by Wauplin — APPROVED** [review]

**Review by julien-c — APPROVED** [review]

File: docs/hub/spaces-config-reference.md
Comment (julien-c) [inline docs/hub/spaces-config-reference.md:38]: is it clear enough that those are TPUs? ^

Comment (apolinario) [inline docs/hub/spaces-config-reference.md:38]:
```suggestion
Value must be a valid hardware flavor. Current valid hardware flavors:
- CPU: `"cpu-basic"`, `"cpu-upgrade"`
- GPU: `"t4-small"`, `"t4-medium"`, `"l4x1"`,
	`"l4x4"`, `"a10g-small"`, `"a10g-large"`, `"a10g-largex2"`,
	`"a10g-largex4"`,`"a100-large"`
- TPU: `"v5e-1x1"`, `"v5e-2x2"`, `"v5e-2x4"`
```
@julien-c I agree it is not clear that those are TPUs, here's an alternative formatting suggestion for this part of the docs

Comment (julien-c) [reply docs/hub/spaces-config-reference.md:38]: yes i agree that's better! 

btw @christophe-rannou maybe we could have encoded tpu inside the strings `"tpu-v5e-1x1"` etc

Comment (osanseviero) [reply docs/hub/spaces-config-reference.md:38]: Yes this is nicer

**Pull request merged**
